pwa/src/templates/templateParts/header/HeaderTemplate.module.css

```diff
@@ -155,6 +149,7 @@ li .denhaag-link__icon {
   }
 
   .primaryNavContainer {
+    padding-block: 1em;
```

Comment:
I think we should use a design-token for this, if we do not have one, you can create one.